src/AWSCredentials.jl

```diff
@@ -154,6 +154,7 @@ function AWSCredentials(;profile=nothing)
 end
 
 
+const ims_local_hostname = Ref{Any}(nothing)
```

Comment:
Since you didn't mark this as `WIP`, I assume this is left over from some testing and should be removed, as well as lines `211-229`.

Reply:
I removed it.  I was looking into using the Instance Metadata Service url, but when it fails (such as on machines that aren't ec2's), it takes forever.  So I commented out most of the code for it and I had left this line.  It's gone now.